### DISCUSSION TITLE
What's the procedure for replacing a name shown on DefiLlama?

### DISCUSSION THREAD

**Discussion by Dizz02** [DefiLlama/DefiLlama-Adapters#7204] [General]
I would like to replace FuzioNetwork DEX with just Fuzio instead but not sure who i can contact. Thank you and nice work by the way.

**Comment by g1nt0ki** [comment]
hi @Dizz02 you can post the new name here

**Reply by Dizz02** [reply]
Fuzio

**Comment by Dizz02** [comment]
Current Name: FuzioNetwork DEX 

New name: Fuzio



Thank you

**Comment by Define101** [comment]
Thanks for letting us know
Should be live within 1 hour
